Scripts/run-tests.ps1

```diff
@@ -21,6 +21,14 @@ $targets = [ordered]@{
     "standalone" = "Tests.Standalone"
 }
 
+$ilverify = FindProgram("ilverify")
+if ($null -eq $ilverify) {
+    &dotnet tool install --global dotnet-ilverify
+    $ilverify = FindProgram("ilverify");
+}
+
```

Comment:
This is really neat :)

Reply:
yep fortunately same thing works on linux, so all good.